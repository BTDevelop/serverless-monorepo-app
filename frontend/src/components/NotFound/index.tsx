import * as React from 'react';
import { Segment } from 'semantic-ui-react';

const NotFound = () => <Segment>Sorry, this page does not exist.</Segment>;

export default NotFound;
