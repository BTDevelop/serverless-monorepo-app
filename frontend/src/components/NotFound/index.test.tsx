import * as React from 'react';
import * as renderer from 'react-test-renderer';
import NotFound from './';

describe('NotFound Component', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
