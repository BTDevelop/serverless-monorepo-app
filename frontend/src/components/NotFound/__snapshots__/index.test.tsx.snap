// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component should render correctly 1`] = `
<div
  className="ui segment"
>
  Sorry, this page does not exist.
</div>
`;
